### PR TITLE
FOOD

### PR DESCRIPTION
**Pull request by plaindocs**
This is kinda hidden away, I can either pull it out into a FOOD menu entry, or we can just mention that we've added it in the next (last) mail out.

### CODE REVIEW

**Comment by plaindocs** [comment]
It is pretty ugly, but it is there.

File: docs/conf/na/2017/venue.rst [hunk 1]
```diff
@@ -33,7 +33,138 @@ All transit in Portland announce stops verbally in English. Most buses have a vi
...
 ********************
 
-We aim to meet all dietary requirements. You will be able to specify these when purchasing your ticket.
+We aim to meet all dietary requirements. Our menu and ingredients are described
+below, although one or two items are still subject to change.
+
+Sunday
+===========
+
+08:00 - 10:00 Wrap buffet
```
Comment (ericholscher) [inline docs/conf/na/2017/venue.rst:42]: These should probably just say `Morning Snack, Lunch, Afternoon Snack`.

**Comment by ericholscher** [comment]
Looks great, just a small tweak to clean it up -- can also just ship this if nobody gets to it.

**Comment by plaindocs** [comment]
I'll ship it and come back to fix if I have time, which is not likely.

**Pull request merged**